frontend/src/App.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from "react";
+import Form from "./components/Form";
+import Title from "./components/Title";
+import Results from "./components/Results";
+import "./index.css";
+import {
+  User,
+  UserSearchRequest,
+  UserSearchResponse
+} from "./protofiles/usersearch_pb.js";
+import { UserSearchServiceClient } from "./protofiles/usersearch_grpc_web_pb";
+import { withRouter } from "react-router";
+import PropTypes from "prop-types";
+
+// eslint-disable-next-line
+
+class App extends Component {
+  state = {
+    users: []
+  };
+
+  handleSubmit = userName => {
+    const userSearch = new UserSearchServiceClient("http://localhost:4000");
+    const request = new UserSearchRequest();
+
+    request.setUserName(userName);
+    userSearch.getAllUsersLikeUsername(request, {}, (err, response) => {
+      console.log(err);
```

Comment:
```suggestion
      if (err){
        console.log(err);
        return
      }
      
```

When we get an error we don't want to trigger the code where we use the response